bookshelf/1-cloud-run/pom.xml

```diff
@@ -121,17 +121,19 @@ Copyright 2019 Google LLC
     <!-- Optional - for hot reload of the web application when using an IDE Eclipse / IDEA -->
     <outputDirectory>${project.build.directory}/${project.build.finalName}/WEB-INF/classes
     </outputDirectory>
-    <resources>
-      <resource>
-        <directory>${project.basedir}/src/main/resources</directory>
-        <filtering>true</filtering>
-      </resource>
-    </resources>
     <plugins>
       <plugin>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-maven-plugin</artifactId>
         <version>${jetty.version}</version>
+        <configuration>
+          <systemProperties>
+            <systemProperty>
+              <name>bookshelf.bucket</name>
```

Comment:
I think I mentioned this in the other PR, but have you considered using env vars over system properties to configure samples? The downside to using systemProperties is that they must be configured as compile time, while environment variables can be compiled whenever.

Reply:
Tried using the below, but the jetty server doesn't pick it up, even if I change `System.getProperty` to `System.getenv`. Is there another configuration setting I can use?

```
        <configuration>
          <env>
            <bookshelf.bucket>${bookshelf.bucket}</bookshelf.bucket>
          </env>
        </configuration>
```

Reply:
I'm not sure you can configure environment variables from a pom.xml.

You should just need to update references like `System.getProperty('bookshelf.bucket')` to `System.getenv('BOOKSHELF_BUCKET')`. Then just set them in terminal with `export BOOKSHELF_BUCKET="your-bookshelf-bucket"` before running the code or configure them via Cloud Run (I think via command line or the GUI).

Reply:
All the search results I can find looking for Jetty maven plugin environment variable use point to using system properties :|

Reply:
Oh, if you configure these inside the the `maven-jetty-plugin` section of your pom, then they will only apply when you use the plugin to run your application (ei, `jetty:run`). [According to the plugin docs](https://www.eclipse.org/jetty/documentation/9.4.x/jetty-maven-plugin.html#jetty:run-forked), `env` configurations only apply when you use `jetty:run-forked`.

When running locally, you should set the environment variables in your shell before executing (e.i. `export BOOKSHELF_BUCKET="your-bookshelf-bucket"`).

If running in docker, you have to set the environment variables in the command (`docker run --env VAR1=value1`).

For Cloud Run, [you configure them during deploy](https://cloud.google.com/run/docs/configuring/environment-variables) and it will set them in the docker container at runtime. 

Here's a [breakdown between system properties and environment variables](https://www.baeldung.com/java-system-get-property-vs-system-getenv).

Reply:
Setting env vars via `export` isn't working